gandalf.js

```diff
@@ -4,125 +4,122 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.